src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+    for scope in ['sb subscription get', 'servicebus subscription delete']:
+        with self.argument_context(scope) as c:
+            c.argument('namespacename', options_list=['--namespace-name'], required=True, help='name of the Namespace'),
+            c.argument('topic_name', options_list=['--topic-name'], help='name of the Topic'),
+            c.argument('subscription_name', options_list=['--name'], help='name of the Subscription of Topic')
+
+
+    with self.argument_context('sb subscription list') as c:
+        c.argument('namespace_name', options_list=['--namespace-name'], required=True, help='name of the Namespace'),
+        c.argument('topic_name', options_list=['--topic-name'], required=True, help='name of the Topic')
+
+###### Region Subscription Rules
+
+# Rules Create
+
+    with self.argument_context('sb rule create') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True),
+        c.argument('namespace_name', options_list=['--namespace-name'], required=True, help='name of the Namespace'),
+        c.argument('topic_name', options_list=['--topic-name'], required=True, help='Topic Name'),
+        c.argument('subscription_name', options_list=['--subscription-name'], required=True, help='Subscription Name'),
+        c.argument('rule_name', options_list=['--name', '-n'], required=True, help='Rule Name'),
+        c.argument('action_sql_expression', options_list=['--action-sql-expression'], help='Action SQL expression.'),
+        c.argument('action_compatibility_level', options_list=['--action-compatibility-level'], type=int,
+                   help='This property is reserved for future use. An integer value showing the compatibility'
+                        ' level, currently hard-coded to 20.'),
+        c.argument('action_requires_preprocessing', type=bool, default=True,
+                   options_list=['--action-requires-preprocessing'],
+                   help='Value that indicates whether the rule action requires preprocessing.'),
+        c.argument('filter_sql_expression', options_list=['--filter-sql-expression'], help='SQL expression. e.g.'),
+        c.argument('filter_requires_preprocessing', type=bool, default=True,
+                   options_list=['--sql-requires-preprocessing'],
+                   help='Value that indicates whether the rule action requires preprocessing.'),
+        c.argument('correlation_id', options_list=['--correlation-id'], help='Identifier of the correlation.'),
+        c.argument('message_id', options_list=['--message-id'], help='Identifier of the message.'),
+        c.argument('to', options_list=['--to'], help='Address to send to.'),
+        c.argument('reply_to', options_list=['--reply_to'], help='Address of the queue to reply to.'),
+        c.argument('label', options_list=['--label'], help='Application specific label.'),
+        c.argument('session_id', options_list=['--session-id'], help='sessionId'),
```

Comment:
`sessionId` is not a helpful help message.

Reply:
updated with help message from swagger